evm-contracts/src/v0.6/dev/PrepaidAggregator.sol

```diff
@@ -146,12 +150,19 @@ contract PrepaidAggregator is AggregatorInterface, Owned, WithdrawalInterface {
     onlyUnenabledAddress(_oracle)
   {
     require(oracleCount() < 42, "cannot add more than 42 oracles");
+    require(_admin != address(0), "admin address must not be 0x0");
+    require(
+      oracles[_oracle].admin == address(0) || oracles[_oracle].admin == _admin,
```

Comment:
The second case after the `||` would still allow the owner to add an oracle twice if the first one was added with the same `_admin` address. I think we should prevent that so that an oracle cannot be added twice. While this would only update the `oracles` mapping, it would duplicate the oracle in `oracleAddresses`.

Reply:
@lorenzb explained to me that the second case after the `||` is to prevent the owner from removing an oracle and then re-adding them with a different admin address.